README.md

```diff
@@ -13,15 +13,17 @@ methods arguments and return values are strictly typed.
 **Constructor:**
 
 ```javascript
-import AWS from 'aws-sdk';
-import DynamoDbDao from '@jupiterone/dynamodb-dao';
+import { DynamoDBClient } from '@aws-sdk/client-dynamodb';
+import { DynamoDBDocumentClient } from '@aws-sdk/lib-dynamodb';
 
-const dynamodb = new AWS.DynamoDB({
+const ddb = new DynamoDBClient({
   apiVersion: '2012-08-10'
-});
+})
 
-const documentClient = new AWS.DynamoDB.DocumentClient({
-  service: dynamodb
+const documentClient = DynamoDBDocumentClient.from(ddb, {
+  marshallOptions: {
+    removeUndefinedValues: true
```

Comment:
AFAIK this recreates the behavior of `v2`. Of all the things in this PR this is the one I'm most concerned about as far as breaking behavior. However, it seems `removeUndefinedValues: true` recreates what was the default behavior in v2:
https://github.com/aws/aws-sdk-js-v3/issues/1816